package.json

```diff
@@ -17,8 +17,8 @@
     "hashtrack": "^1.0.2",
     "underscore": "^1.7.0",
     "jquery-form": "~3.50.0",
-    "OverlappingMarkerSpiderfier": "yagoferrer/OverlappingMarkerSpiderfier#747476d7bf",
-    "Flot": "flot/flot#453b017cc5"
+    "marker-spider": "0.1.0",
```

Comment:
All the other versions seem to have punctuation prefixes.  I assume they mean something... do we need them on these two packages?